src/components/SignUp/SignUp.js

```diff
@@ -68,11 +71,13 @@ export class SignUp extends React.Component {
         });
 
         setTimeout(() => {
-          this.setState({
-            statusMessage: 'enter_password',
-          });
+          if (this.isComponentMounted) {
```

Comment:
Introduced this because the timeout callback could get executed after the component was unmounted, which was throwing an error in one of the tests